### DISCUSSION TITLE
something like https://github.com/chaoren/vim-wordmotion?

### DISCUSSION THREAD

**Discussion by bkerin** [akinomyoga/ble.sh#301] [Ideas]
In my current round of going through and trying to fix up little things that have annoyed me for years I found a few vim modules:

    Plug 'chaoren/vim-wordmotion'  (no integration yet)
    Plug 'tpope/vim-surround'          (ble.sh already has integration :) )
    Plug 'A-Ih/argtextobj.vim'           (not integration but not really relevant in bash, could be mapped to word-wise operation)

I'm just checking if by small chance I missed some existing thing on the first one?  If it doesn't exist yet it looks It looks like it would just need new widgets (and bindings) for the Wordwise motion commands documented in the vim section
of the blesh manual, and maybe corresponding stuff for text objects, and I will do this.

**Comment by akinomyoga** [comment]
> I'm just checking if by small chance I missed some existing thing on the first one?

No, `ble.sh` doesn't have an existing implementation on the mentioned `vim-wordmotion` as you observe, so you can take a shot at implementing it by yourself.
